### ISSUE TITLE
CI fails due to deprecated Mambaforge installer

### ISSUE THREAD

**Issue by niketagrawal** [tudat-team/tudat#243]
Latest CI run : https://github.com/tudat-team/tudat/actions/runs/11124374586/job/30909780849

```
Warning: Future Miniforge releases will NOT build Mambaforge installers. We advise you switch to Miniforge at your earliest convenience. More details at [https://conda-forge.org/news/2024/07/29/sunsetting-mambaforge/.](https://conda-forge.org/news/2024/07/29/sunsetting-mambaforge/)
  Warning: ERROR: executing pre_install.sh failed
  
  ERROR: executing pre_install.sh failed
Error: The process '/usr/bin/bash' failed with exit code 1
```

### Root cause

MambaForge installers are deprecated as pointed by the warning message. See https://conda-forge.org/news/2024/07/29/sunsetting-mambaforge/ 
Potential fix is to switch to MiniForge installer for the conda environment setup step in the Ci workflow.

**Comment by niketagrawal** [comment]
The fix mentioned above did not solve the issue. 
https://github.com/tudat-team/tudat/actions/runs/11128997276/job/30925041364

**Comment by niketagrawal** [comment]
Closing as this issue is no longer seen on latest commit.

https://github.com/tudat-team/tudat/actions/runs/11139860103

**Comment by niketagrawal** [comment]
Issue is seen again on https://github.com/tudat-team/tudat/actions/runs/11340844465/job/31537969986

**Comment by niketagrawal** [comment]
Issue thread with some information on this issue: https://github.com/readthedocs/readthedocs.org/issues/11690

**Comment by DominicDirkx** [comment]
@niketagrawal any update on how to resolve this?

**Comment by niketagrawal** [comment]
> @niketagrawal any update on how to resolve this?

In the CI workflow build_and_test.yml, the step Setup conda environment using mamba uses `mamba` (faster alternative to conda) to create an empty conda environment named 'tudat' and activate it. 

Since the mamba installer (Mambaforge) will be deprecated soon, a solution is to switch back to using default conda as shown in the basic usage example in the documentation https://github.com/marketplace/actions/setup-miniconda#example-1-basic-usage 

A draft PR #264  is created to test the changes.

**Comment by niketagrawal** [comment]
Fixed with #264
